XamarinCommunityToolkit/Views/SegmentedView/Android/SegmentedViewRenderer.android.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using Android.Content;
+using Xamarin.CommunityToolkit.Android.UI.Views;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using RadioButton = Android.Widget.RadioButton;
+
+[assembly: ExportRenderer(typeof(SegmentedView), typeof(SegmentedViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.Android.UI.Views
+{
+	public class SegmentedViewRenderer : ViewRenderer<SegmentedView, FormsSegments>
+	{
+		readonly Context context;
```

Comment:
The `ViewRenderer` exposes the `Context` as property, can you remove this?

Reply:
Done